framework/test/mocha/common/lisk-mocha-runner/file_manager.js

```diff
@@ -0,0 +1,46 @@
+const find = require('find');
+
+const testTypesMap = {
+	unit: 'framework/test/mocha/unit/',
+	integration: 'framework/test/mocha/integration/',
+	'functional:ws': 'framework/test/mocha/functional/ws/',
+	'functional:get': 'framework/test/mocha/functional/http/get/',
+	'functional:post': 'framework/test/mocha/functional/http/post/',
```

Comment:
`functional:put` also needs to be added. Also in other places too

Reply:
@shuse2 There is a separate issue for that #3060